app/src/main/java/org/mozilla/fenix/Config.kt

```diff
@@ -60,6 +60,16 @@ enum class ReleaseChannel {
 
     val isFenix: Boolean
         get() = !isFennec
+
+    val deepLinkScheme: String
+        get() = when (this) {
+            FenixNightly -> "fenix-nightly"
```

Comment:
At the time, when we added the deepLinkScheme, the only use case for it was [in the manifest](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/AndroidManifest.xml#L82).

I don't think we can avoid the duplication to set the values in the manifest an in the Kotlin layer, but what do you think about using a `buildConfigField` so we set the values in the same `build.gradle` place? We wouldn't need the `Config.deepLinkScheme` parsing and it would be less likely for errors if we decide to change our variants tomorrow (it's quite likely actually with LeanPlum things 😄 ).

Reply:
Sure, I'll look into creating a `buildConfigField`. Thanks,